utils/build-script-impl

```diff
@@ -1943,21 +1942,13 @@ for host in "${ALL_HOSTS[@]}"; do
                     build_targets=(llvm-tblgen clang-headers)
                 fi
 
-                # Note: we set the variable:
-                #
-                # LLVM_TOOL_SWIFT_BUILD
-                #
-                # below because this script builds swift separately, and people
-                # often have reasons to symlink the swift directory into
-                # llvm/tools, e.g. to build LLDB.
```

Comment:
That setting is needed if you want to allow users to have swift sources checked out or symlinked under LLVM/tools and still use the build script for building.